### PR TITLE
RA-166

### PR DESCRIPTION
**Pull request by aoller**

### CODE REVIEW

**Comment by vguzmanp** [comment]
You have some things that you create and then rename / remove, etc. You should squash all those to have a clean commit history

For example, `ControllerStatus` is created in one commit, and removed in the next one.

File: VAS.Core/Interfaces/IRenderingJobsController.cs [hunk 1]
```diff
@@ -15,30 +15,28 @@
...
+		void CancelJobs (RangeObservableCollection<JobVM> cancelJobs);
 
 		void CancelCurrentJob ();
 
 		void CancelJob (JobVM job);
 
 		void CancelAllJobs ();
 
-		void AddJob (JobVM job);
+		void AddJob (Job job);
```
Comment (aoller) [inline VAS.Core/Interfaces/IRenderingJobsController.cs]: In order to use the same Job object from LongoMatch and RiftAnalyst.

File: VAS.Services/CoreEventsManager.cs [hunk 2]
```diff
@@ -215,7 +215,8 @@ protected void RenderPlay (Project project, TimelineEvent play)
 				element = new PlaylistPlayElement (play);
 				playlist.Elements.Add (element);
 				job = new EditionJob (playlist, settings);
-				App.Current.RenderingJobsManger.AddJob (job);
+				App.Current.RenderingJobsManger?.AddJob (job);
+				App.Current.RenderingJobsController?.AddJob (job);
```
Comment (aoller) [inline VAS.Services/CoreEventsManager.cs]: Yes, and by the same reason why I used Job instead JobVM.

File: VAS.UI.Gtk2/UI/Dialog/RenderingJobsDialog.cs [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+		public RenderingJobsDialogVM ViewModel {
+			get {
+				return viewModel;
+			}
+			set {
+				if (ViewModel != null) {
+					viewModel.PropertyChanged -= HandleViewModelChanged;
+				}
+				viewModel = value;
+				renderingjobstreeview2.SetViewModel (viewModel.JobCollectionVM);
```
Comment (aoller) [inline VAS.UI.Gtk2/UI/Dialog/RenderingJobsDialog.cs]: It is copied from LongoMatch.

File: VAS.Core/Events/JobEvents.cs [hunk 4]
```diff
@@ -26,11 +27,11 @@ public class ClearDoneJobsEvent : Event
 
 	public class RetrySelectedJobsEvent : Event
```
Comment (vguzmanp) [inline VAS.Core/Events/JobEvents.cs]: Modify to `RetryJobsEvent`

**Pull request closed without merging**